src/shared/components/oncoprint/TooltipUtils.ts

```diff
@@ -141,7 +141,7 @@ export function makeHeatmapTrackTooltip(genetic_alteration_type:MolecularProfile
             case AlterationTypeConstants.METHYLATION:
                 data_header = 'METHYLATION: ';
                 break;
-            case AlterationTypeConstants.TREATMENT_RESPONSE:
+            case AlterationTypeConstants.GENERIC_ASSAY:
                 data_header = 'TREATMENT: ';
```

Comment:
Can we get rid of this hard-coded value? It would be great to somehow know the name of an entity that is linked to a profile of type `GenericAssayTypeConstants.TREATMENT_RESPONSE`.